### PR TITLE
Electrocution noises

### PR DESCRIPTION
**Pull request by vulppine**
## About the PR 

Adds noises for when an entity gets electrocuted by another entity.

### CODE REVIEW

File: Content.Server/Electrocution/Components/ElectrifiedComponent.cs [hunk 1]
```diff
@@ -65,5 +66,20 @@ public sealed class ElectrifiedComponent : Component
...
         public float SiemensCoefficient { get; } = 1f;
+
+        [DataField("shockNoises")]
+        public List<SoundSpecifier> ShockNoises { get; } = new()
+        {
+            new SoundPathSpecifier("/Audio/Effects/sparks1.ogg"),
+            new SoundPathSpecifier("/Audio/Effects/sparks2.ogg"),
+            new SoundPathSpecifier("/Audio/Effects/sparks3.ogg"),
+            new SoundPathSpecifier("/Audio/Effects/sparks4.ogg")
+        };
```
Comment (mirrorcult) [inline Content.Server/Electrocution/Components/ElectrifiedComponent.cs:77]: this is what `SoundCollection` is for mate

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Content.Server/Electrocution/Components/ElectrifiedComponent.cs (hunk 1 above)
Comment (mirrorcult) [reply Content.Server/Electrocution/Components/ElectrifiedComponent.cs:77]: to be more clear
```suggestion
        [DataField("shockNoises")]
        public SoundSpecifier ShockNoise { get; } = new SoundCollectionSpecifier("whatever collection ID");
```

Comment (vulppine) [reply Content.Server/Electrocution/Components/ElectrifiedComponent.cs]: whoops

File: Content.Server/Electrocution/Components/ElectrifiedComponent.cs [hunk 2]
```diff
@@ -65,5 +66,14 @@ public sealed class ElectrifiedComponent : Component
 
         [DataField("siemensCoefficient")]
         public float SiemensCoefficient { get; } = 1f;
+
+        [DataField("shockNoises")]
+        public SoundCollectionSpecifier ShockNoises { get; } = new("sparks");
```
Comment (metalgearsloth) [inline Content.Server/Electrocution/Components/ElectrifiedComponent.cs:71]:
```suggestion
        public SoundSpecifier ShockNoises { get; } = new SoundCollectionSpecifier("sparks");
```

**Pull request merged**